macOS/NetworkProtectionSystemExtension/NetworkProtectionSystemExtension_SparkleDistribution.entitlements

```diff
@@ -19,13 +19,6 @@
 	<true/>
 	<key>com.apple.security.network.server</key>
 	<true/>
-	<key>com.apple.security.temporary-exception.files.absolute-path.read-write</key>
-	<array>
-		<string>/Library/Keychains/System.keychain</string>
-		<string>/private/var/db/mds/system/mds.lock</string>
-		<string>/Library/Keychains/</string>
-		<string>/Library/Application Support/</string>
-	</array>
```

Comment:
We got rejected due to these, and they're really not necessary.  This is quite old VPN stuff.